### ISSUE TITLE
[MHV Portal / Accessibility] List of links content provided using navigation/wayfinding elements

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#122468]
### 📝 User story
As a screen reader user, I want to be able to differentiate between navigation/wayfinding elements and page content, so that I know how to interact with the site and consume the content.

### 📝 Problem statement
Lists of links content on homepage is provided with navigation `<nav>` elements.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [My HealtheVet homepage](https://staging.va.gov/my-health)

<img width="970" height="551" alt="Screenshot showing three columns of link lists, each outlined and labeled with a "nav" label" src="https://github.com/user-attachments/assets/4bb8b560-51c8-4916-8d8a-4b7f368896f6" />

---

```
<nav>
<ul class="mhv-c-link-list">
<li><a href="/health-care/get-health-id-card/">Veteran health information card</a></li>
<li><a href="/my-health/update-benefits-information-form-10-10ezr/introduction">Update health benefits info (10-10EZR)</a></li>
<li><a href="/health-care/copay-rates/">Current Veteran copay rates</a></li>
…
</ul>
</nav>

```
#### 👣 Steps to recreate
1. Navigate to the My HealtheVet homepage.
2. Scroll to the section that follows the Need help? heading.
3. Examine the rendered properties of the link lists.
4. Notice the use of the `<nav>` element.

</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure.](https://www.w3.org/WAI/WCAG22/Understanding/adaptable)
  - [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks.](https://design.va.gov/about/experience-standards/#category-usability)
- [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps
Remove the `<nav>` element from the link lists.

### ✅ Acceptance criteria
- [ ] The system displays the lists of links using list markup.
- [ ] The screen reader user hears the link lists announced as a list with [x] items.

### ✅ Possible tasks
- [x] Review, refine and acknowledge feedback.
- [x] Update ticket information (assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

#### General resources
- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)
